api/addons/v1beta1/conversion.go

```diff
@@ -122,6 +123,24 @@ func Convert_v1beta2_ResourceSetBinding_To_Pointer_v1beta1_ResourceSetBinding(in
 	return autoConvert_v1beta2_ResourceSetBinding_To_v1beta1_ResourceSetBinding(in, *out, s)
 }
 
+func Convert_v1beta1_ResourceBinding_To_v1beta2_ResourceBinding(in *ResourceBinding, out *addonsv1.ResourceBinding, s apimachineryconversion.Scope) error {
+	if err := autoConvert_v1beta1_ResourceBinding_To_v1beta2_ResourceBinding(in, out, s); err != nil {
+		return err
+	}
+	if in.LastAppliedTime != nil && !reflect.DeepEqual(in.LastAppliedTime, &metav1.Time{}) {
+		out.LastAppliedTime = *in.LastAppliedTime
+	}
+	return nil
+}
+
+func Convert_v1beta2_ResourceBinding_To_v1beta1_ResourceBinding(in *addonsv1.ResourceBinding, out *ResourceBinding, s apimachineryconversion.Scope) error {
+	if err := autoConvert_v1beta2_ResourceBinding_To_v1beta1_ResourceBinding(in, out, s); err != nil {
+		return err
+	}
+	out.LastAppliedTime = ptr.To(in.LastAppliedTime)
```

Comment:
Should this be 
```suggestion
	if !reflect.DeepEqual(in.LastAppliedTime, &metav1.Time{}) {
		out.LastAppliedTime = ptr.To(in.LastAppliedTime)
	}
```
Otherwise we might have ptr to `metav1.Time{}` which seems wrong

Reply:
same for all the v1beta2-->X conversions

Reply:
Nice catch. You're right. Another gap in the fuzzer. Apparently we never get an empty metav1.Time as input

Reply:
I fixed the metav1.Time fuzzer, we have a similar situation with IntOrString. I'll fix that one in  #12525. Somehow I got failures in that error with a fix for the IntOrString fuzzer